scripts/update/radarr.sh

```diff
@@ -44,8 +44,17 @@ if [[ -f /install/.radarr.lock ]]; then
             fi
 
         elif [[ $(_radarr_version) = "mono-v2" ]]; then
-            echo_warn "Please upgrade your radarr to v3. It will be migrated to .Net core on the next \`box update\` run"
+            echo_warn "Radarr v0.2 is EOL and not supported. Please upgrade your radarr to v3. It will be migrated to .Net core on the next \`box update\` run"
```

Comment:
Might as well make this clear while we're here
```suggestion
            echo_warn "Radarr v0.2 is EOL and not supported. Please upgrade your radarr to v3. An attempt will be made to migrate to .Net core on the next \`box update\` run"
```